pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/SegmentTarPushJob.java

```diff
@@ -85,7 +87,7 @@ public void pushOneTarFile(FileSystem fs, Path path) throws Exception {
         fileName = fileName.split(".tar")[0];
         LOGGER.info("******** Upoading file: {} to Host: {} and Port: {} *******", fileName, host, _port);
         try {
-          int responseCode = FileUploadUtils.sendSegmentFile(host, _port, fileName, inputStream, length);
+          int responseCode = sendSegmentFile(host, _port, fileName, fs, path, length);
```

Comment:
Remove the 'length' and 'inputStream', we should get them from sengSegmentFile() method.